NuKeeper.Abstractions/Configuration/GitPullRequestMergeStrategy.cs

```diff
@@ -0,0 +1,10 @@
+namespace NuKeeper.Abstractions.Configuration
+{
+    public enum GitPullRequestMergeStrategy
+    {
+        noFastForward,
```

Comment:
Please use PascalCase and make sure that the default has a value of 0

Reply:
Will do